src/objects/RecurringDonationChangeHistory__c.object

```diff
@@ -0,0 +1,389 @@
+        <label>New Amount</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Currency</type>
+    </fields>
+    <fields>
+        <fullName>New_Campaign__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The new Campaign for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Campaign for the related Recurring Donation.</inlineHelpText>
+        <label>New Campaign</label>
+        <referenceTo>Campaign</referenceTo>
+        <relationshipLabel>Recurring Donation Change History (New Campaign)</relationshipLabel>
+        <relationshipName>RD_Change_History_New_Campaign</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>New_Payment_Method__c</fullName>
+        <description>The new Payment Method for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Payment Method for the related Recurring Donation.</inlineHelpText>
+        <label>New Payment Method</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>New_Schedule__c</fullName>
+        <description>The new payment schedule for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new payment schedule for the related Recurring Donation.</inlineHelpText>
+        <label>New Schedule</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>New_Status_Reason__c</fullName>
+        <description>The new Status Reason for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Status Reason for the related Recurring Donation.</inlineHelpText>
+        <label>New Status Reason</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>New_Status__c</fullName>
+        <description>The new Status for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Status for the related Recurring Donation.</inlineHelpText>
+        <label>New Status</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
We should be aware that as a plain text field this will contain the Status value in the language of the User that made the change (assuming the code retrieves the Label for the picklist api name). It won't dynamically translate like other picklist values. Making this a picklist field (as well as StatusReason and any others) adds a lot of complexity as well, so keeping with a plain text field isn't bad. But it's good to document the translation limitation.